locale/de_DE/locale.xml

```diff
@@ -247,7 +247,7 @@
 	<message key="grid.category.parentCategory">Übergeordnete Kategorie</message>
 	<message key="grid.category.removeText">Sind Sie sicher, dass Sie diese Kategorie entfernen möchten?</message>
 	<message key="grid.category.nameRequired">Bitte geben Sie einen Namen für die Kategorie ein.</message>
-	<message key="grid.category.categoryDetails">Kategoriedetails</message>
+	<message key="grid.category.categoryDetails">Genauere Angaben zur Kategorie</message>
```

Comment:
"details" has been translated to "Details" throughout the OJS and OMP translation. I am a little reluctant to make a single change that breaks consistency.